composer.json

```diff
@@ -25,7 +25,8 @@
         "sebastian/diff": "~1.1"
     },
     "require-dev": {
-        "satooshi/php-coveralls": "0.7.*@dev"
+        "phpunit/phpunit": "^4.5|^5",
```

Comment:
:-1: If we are able to not make it dependents, don't do it.

This will install a **lot** of dependencies, making composer longer to resolve.

Plus, all developer does not work with vendor/bin/phpunit.

What about directly download the `.phar` on travis?

Example: https://github.com/sonata-project/dev-kit/blob/a4574f4ff3428bdd89ecf1c2f4f0190227abb250/project/.travis/install_test.sh#L4-L13